scripts/create-container.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Flags
+# set -e
+
+# External files
+# Get cfg values
+source "$PWD/scripts/config/lxc.cfg"
+
+# Check config file
+echo "Checking config file"
+if [ ! -e "$LXC_CONFIG" ] ; then
+  echo "Creating config file: $LXC_CONFIG"
+  network_link="$(brctl show | awk '{if ($1 != "bridge")  print $1 }')"
+  cat >"$LXC_CONFIG" <<EOL
+# Network configuration
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.link = $network_link
+EOL
+fi
+
+# Print configuration
+echo "* CONFIGURATION:"
+echo "  - Name: $NAME"
+echo "  - Template: $TEMPLATE"
+echo "  - LXC Configuration: $LXC_CONFIG"
+echo "  - Release: $RLS"
+echo "  - Host: $HOST"
+echo "	- Project Name: $PROJECT_NAME"
+echo "	- Project Directory: $PROJECT_PATH"
+echo
+
+echo
+echo
+
+# Check container
+exist_container="$(sudo lxc-ls "$NAME")"
+if [ -z "${exist_container}" ] ; then
+  echo "Creating container $NAME"
+  sudo lxc-create --name "$NAME" -f "$LXC_CONFIG" -t "$TEMPLATE" -l INFO --logfile "./log/$NAME-create.log" -- --release "$RLS"
+fi
+echo "Container ready"
+
+# Check if container is running, if not start
+count="0"
+while [ "$count" -lt 5 ] && [ -z "$is_running" ]; do
+  is_running=$(sudo lxc-ls --running -f | grep "$NAME")
+  if [ -z "$is_running" ] ; then
+    echo "Starting container"
+    sudo lxc-start -n "$NAME" -d -l INFO --logfile "./log/$NAME-start.log"
+    ((count++))
+  fi
+done
+
+# If not is running stop execution
+if [ -z "$is_running" ]; then
+  echo "Container not started..."
+  echo "STOP EXECUTION"
+  exit 0
+fi
+
+echo "Container is running..."
+# Wait to start container and check the ip
+count="0"
+ip_container="$( sudo lxc-info -n "$NAME" -iH )"
+while [ "$count" -lt 5 ] && [ -z "$ip_container" ] ; do
+  sleep 2
+  echo "waiting for container ip..."
+  ip_container="$( sudo lxc-info -n "$NAME" -iH )"
+  ((count++))
+done
+echo "Container IP: $ip_container"
+echo
+
+# ADD IP TO HOSTS
+echo "Remove old host $HOST from /etc/hosts"
+sudo sed -i '/'"$HOST"'/d' /etc/hosts
+host_entry="$ip_container       $HOST"
+echo "Add '$host_entry' to /etc/hosts"
+sudo -- sh -c "echo $host_entry >> /etc/hosts"
+echo
+# SSH Key
```

Comment:
can we add some spacing between these blocks as we did until this one? below here the script it's a bit harder to read.